docs/CONTRIBUTING.md

```diff
@@ -64,11 +70,35 @@ t.Run("with nested changes trigger", func (t *testing.T) {
 
 **Note**: After your PR has been merged, and the feature either reaches general availability, you should remove the `skipIfBeta()` flag.
 
-## Best Practices for Adding a New Endpoint
+## Adding New Endpoints
+
+### Scaffolding a Resource
+
+When creating a new resource you can use the helper script `generate_resource` to quickly setup boilerplate code for adding a new set of endpoints related to that resource:
+
+#### Running the script directly
+```sh
+cd ./scripts/generate_resource
+go run . example_resource
+```
+
+#### Running the Makefile target `generate`
+```sh
+RESOURCE=example_resource make generate
```

Comment:
This command is different from what is in the PR description. Do both options work?

Reply:
Just updated!